inc/commonitilvalidation.class.php

```diff
@@ -1405,7 +1405,7 @@ static function computeValidationStatus(CommonITILObject $item) {
       if ($validation_percent > 0) {
          if (($statuses[self::ACCEPTED]*100/$total) >= $validation_percent) {
             $validation_status = self::ACCEPTED;
-         } else if (($statuses[self::REFUSED]*100/$total) >= $validation_percent) {
+         } else if (($statuses[self::REFUSED]*100/$total) + $validation_percent > 100) {
```

Comment:
Maybe it would b more comprehensible to write `{refusal %} > 100% - {minimum_validation %}` (i.e. `10% > 100% - 100%`.
```suggestion
         } else if (($statuses[self::REFUSED]*100/$total) > 100 - $validation_percent) {
```